Fix Javadoc warnings

### Describe the bug

There are javadoc warnings

### Expected behavior

_No response_

### Minimal reproducible example

_No response_

### Add-on Version

1.0.2-SNAPSHOT

### Vaadin Version

-

### Additional information

```
[WARNING] Javadoc Warnings
[WARNING] J:\fc\addons\DayOfWeekSelectorAddon\src\main\java\com\flowingcode\vaadin\addons\dayofweekselector\DayOfWeekSelector.java:195: warning: @return tag cannot be used in method with void return type.
[WARNING] public void setWeekDaysShort(List<String> weekdaysShort) {
[WARNING] ^
[WARNING] J:\fc\addons\DayOfWeekSelectorAddon\src\main\java\com\flowingcode\vaadin\addons\dayofweekselector\DayOfWeekSelector.java:212: warning: @param argument "firstDayOfWeek" is not a parameter name.
[WARNING] * @param firstDayOfWeek the index of the first day of the week
[WARNING] ^
[WARNING] J:\fc\addons\DayOfWeekSelectorAddon\src\main\java\com\flowingcode\vaadin\addons\dayofweekselector\DayOfWeekSelector.java:216: warning: @return tag cannot be used in method with void return type.
[WARNING] public void setFirstDayOfWeek(DayOfWeek first) {

--- a/pom.xml
+++ b/pom.xml
@@ -305,7 +305,7 @@
                         <configuration>
                             <quiet>true</quiet>
                             <doclint>none</doclint>
-                            <failOnWarnings>false</failOnWarnings>
+                            <failOnWarnings>true</failOnWarnings>
                             <links>
                                 <link>https://javadoc.io/doc/com.vaadin/vaadin-platform-javadoc/${vaadin.version}</link>
                             </links>

--- a/src/main/java/com/flowingcode/vaadin/addons/dayofweekselector/DayOfWeekSelector.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/dayofweekselector/DayOfWeekSelector.java
@@ -190,7 +190,6 @@ public class DayOfWeekSelector extends CustomField<Set<DayOfWeek>> {
    * Sets the short names of the week days, starting from {@code sun} and ending on {@code sat}.
    *
    * @param weekdaysShort the short names of the week days
-   * @return this instance for method chaining
    */
   public void setWeekDaysShort(List<String> weekdaysShort) {
     Objects.requireNonNull(weekdaysShort);
@@ -209,8 +208,7 @@ public class DayOfWeekSelector extends CustomField<Set<DayOfWeek>> {
    * 0 for Sunday, 1 for Monday, 2 for Tuesday, 3 for Wednesday, 4 for Thursday, 5 for Friday, 6 for
    * Saturday.
    *
-   * @param firstDayOfWeek the index of the first day of the week
-   * @return this instance for method chaining
+   * @param first the index of the first day of the week
    * @throws IllegalArgumentException if firstDayOfWeek is invalid
    */
   public void setFirstDayOfWeek(DayOfWeek first) {

